_sass/_choose.scss

```diff
@@ -90,6 +90,17 @@
     font-size: 1.25rem;
   }
 
+  .card-body {
+    padding-left: 0.25rem;
+    padding-right: 0.25rem;
+
+
+    @media (min-width: 768px) {
+      padding-left: var(--bs-card-spacer-x);
+      padding-right: var(--bs-card-spacer-x);
+    }
+  }
```

Comment:
This fixes an issue with contents exceeding the width of the card on mobile devices